### PR TITLE
Document DP limitations and known issues

### PR DESCRIPTION
**Pull request by markspolakovs**

### CODE REVIEW

File: docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -15,12 +23,31 @@ We essentially need to support two fairly distinct types of user:
...
+  Best efforts will be made but the intention is this iterates often and no backwards compatibility is provided.
+  We will show how to migrate from X-1 to X but no more than that, users should be following an agile lifecycle of constant upgrade.
+* The Couchbase cluster monitor is proprietary and requires access to the repository to build it into the container.
+  The container can be built without it by removing it and there is a https://github.com/couchbaselabs/observability/tree/main/tools/build-oss-container.sh[helper script^] that does this, however certain features (in particular Grafana dashboards) may not be fully functional without it.
+  We plan to address this in a future release.
+
+== Known Issues
+
+CMOS is provided as a Developer Preview and may have bugs, missing features, or other issues.
+Below is an incomplete of issues that we plan to address in a future release.
```
Comment (maria-robobug) [inline docs/modules/ROOT/pages/index.adoc:39]:
```suggestion
Below is an incomplete list of issues that we plan to address in a future release.
```

File: docs/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -15,12 +23,31 @@ We essentially need to support two fairly distinct types of user:
...
+To see the full list or report any issues you encounter. please check the JIRA board and Forums linked below.
```
Comment (maria-robobug) [inline docs/modules/ROOT/pages/index.adoc:40]:
```suggestion
To see the full list or report any issues you encounter please check the JIRA board and Forums linked below.
```

**Review by maria-robobug — APPROVED** [review]
👍 Minor comments.

**Review by redm2 — APPROVED** [review]

File: docs/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -15,12 +23,32 @@ We essentially need to support two fairly distinct types of user:
...
-* The Couchbase cluster monitor is proprietary and requires access to the repository to build it into the container. The container can be built without it by removing it and there is a https://github.com/couchbaselabs/observability/tree/main/tools/build-oss-container.sh[helper script^] that does this.
+* CMOS is built for Couchbase Server 7.0 and above.
+  Lower versions are supported but some features might be missing.
+  We intend to improve support for lower versions in future releases, however Couchbase Server 7.0 and above will remain the primary target of development.
+* Limited cross-version compatibility by supporting migrating from previous version to latest version.
+  Best efforts will be made but the intention is this iterates often and no backwards compatibility is provided.
+  We will show how to migrate from X-1 to X but no more than that, users should be following an agile lifecycle of constant upgrade.
+* The Couchbase cluster monitor is proprietary and requires access to the repository to build it into the container.
```
Comment (markspolakovs) [inline docs/modules/ROOT/pages/index.adoc:32]:
```suggestion
* Part of CMOS is the closed-source Couchbase Cluster Monitor, which is included from a private repository at build time.
```

@ingenthr @anilkumar29 Wording change per our chat earlier - how does this sound?

Comment (anilkumar29) [reply docs/modules/ROOT/pages/index.adoc:32]: CMOS is open source; however, the `Couchbase Cluster Monitor`  is a closed source (private repo) component. If you are building from the source, this component will require access to the repository to make it into the container.

Comment (markspolakovs) [reply docs/modules/ROOT/pages/index.adoc:32]:
```suggestion
* CMOS is open source; however, the Couchbase Cluster Monitor is a closed source (private repository) component.
  If you are building from source, this component will require access to the repository to be included in the container.
```

**Pull request merged**